storage/redis.go

```diff
@@ -16,10 +19,18 @@ import (
 
 // ------------------- REDIS CLUSTER STORAGE MANAGER -------------------------------
 
-var redisClusterSingleton redis.UniversalClient
-var redisLogPrefix = "redis"
-var ENV_REDIS_PREFIX = "TYK_PMP_REDIS"
-var ctx = context.Background()
+var (
+	redisClusterSingleton *RedisManager
+	redisLogPrefix        = "redis"
+	envRedisPrefix        = "TYK_PMP_REDIS"
+	ctx                   = context.Background()
+)
+
+type RedisManager struct {
```

Comment:
@mativm02 let's change this name to StorageManager or StorageHandler ( we use that one MDCB)